apis/cache/v1alpha1/cache_subnet_group_types.go

```diff
@@ -1,12 +1,9 @@
 /*
-Copyright 2019 The Crossplane Authors.
-
+Copyright 2020 The Crossplane Authors.
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
```

Comment:
It appears these license headers were unintentionally modified throughout the PR